docs/data-sources/deep_merge_yaml.md

```diff
@@ -6,21 +6,13 @@ description: |-
   The deep_merge_yaml data source accepts a list of YAML strings as input and deep merges into a single YAML string as output.
 ---
 
-# Data Source `utils_deep_merge_yaml`
+# utils_deep_merge_yaml (Data Source)
 
 The `deep_merge_yaml` data source accepts a list of YAML strings as input and deep merges into a single YAML string as output.
 
 ## Example Usage
 
 ```terraform
-terraform {
-  required_providers {
-    utils = {
-      source = "cloudposse/utils"
-    }
-  }
-}
```

Comment:
Why remove this ?

Reply:
this is not me, the tool did it, prob new version of it